smarts/core/sumo_road_network.py

```diff
@@ -150,15 +150,19 @@ def from_spec(cls, map_spec: MapSpec):
         # Connections to internal lanes are implicit. If `withInternal=True` is
         # set internal junctions and the connections from internal lanes are
         # loaded into the network graph.
+        # pytype: disable=module-attr
         G = sumolib.net.readNet(net_file, withInternal=True)
+        # pytype: enable=module-attr
 
         if not cls._check_net_origin(G.getBoundary()):
             shifted_net_file = cls.shifted_net_file_path(net_file)
             if os.path.isfile(shifted_net_file) or (
                 map_spec.shift_to_origin
                 and cls._shift_coordinates(net_file, shifted_net_file)
             ):
+                # pytype: disable=module-attr
                 G = sumolib.net.readNet(shifted_net_file, withInternal=True)
+                # pytype: enable=module-attr
```

Comment:
Similar suggestion, as previous, to use a single line pytype disable comment.
```suggestion
                G = sumolib.net.readNet(shifted_net_file, withInternal=True)  # pytype: disable=module-attr
```